omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/RestTestConstants1_9.java

```diff
@@ -1,44 +1,46 @@
-package org.openmrs.module.webservices.rest.web;
-
-public class RestTestConstants1_9 {

```

Comment:
Something went wrong, probably with formatting / line spacing. What actually changed here, and is it possible to avoid modifying the entire file to add a line?